packages/frontend-shared/src/gql-components/topnav/VerticalBrowserListItems.vue

```diff
@@ -123,7 +121,12 @@ const props = withDefaults(defineProps <{
 })
 
 const browsers = computed(() => {
-  return (props.gql.browsers ?? []).slice().sort((a, b) => a.displayName > b.displayName ? 1 : -1)
+  const alphaSortedBrowser = (props.gql.browsers ?? []).slice().sort((a, b) => a.displayName > b.displayName ? 1 : -1)
```

Comment:
maybe a `sortBy` makes this more readable? 
```suggestion
  const alphaSortedBrowser = _.sortBy(props.gql.browsers ?? [], 'displayName')

```